services/nuget/nuget-v3-service-family.js

```diff
@@ -139,6 +133,21 @@ function createServiceFamily({
       return renderVersionBadge(props)
     }
 
+    /*
+     * Extract version information from the raw package info.
+     */
+    transform({ json, includePrereleases }) {
+      if (json.data.length === 1 && json.data[0].versions.length > 0) {
+        const packageInfo = json.data[0]
+        const versions = packageInfo.versions.map(item =>
```

Comment:
Don't feel obligated to change anything, but just noting for future reference we could utilize destructuring in these cases
e.g.
```suggestion
        const { versions: packageVersions } = json.data[0]
        const versions = packageVersions.map(item =>
```